tests/pvc_test.go

```diff
@@ -323,6 +324,17 @@ func createPodUsingPVCWithCommand(namespace, name string, pvc *corev1.Persistent
 							MountPath: "/data",
 						},
 					},
+					SecurityContext: &corev1.SecurityContext{
+						AllowPrivilegeEscalation: ptr.To[bool](false),
+						Capabilities: &corev1.Capabilities{
+							Drop: []corev1.Capability{"ALL"},
+						},
+						RunAsNonRoot: ptr.To[bool](true),
+						SeccompProfile: &corev1.SeccompProfile{
+							Type: corev1.SeccompProfileTypeRuntimeDefault,
+						},
+						Privileged: ptr.To[bool](false),
+					},
```

Comment:
how come this is only needed now? PSA has been a must for quite some time

Reply:
Its not really needed, just never did it, so I get warnings.